Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -66,6 +76,10 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         
         request.offerPayLater = payLaterToggle.isOn
 
+        if origamiCheckoutToggle.isOn {
```

Comment:
Take it or leave it - but instead of doing the whole processInfoArgs & Root.plist song and dance, we could just create a new `BTAPIClient` and `BTPayPalClient` w/ the specified tokenization key if this toggle is turned on, all within the ViewController.

I don't feel strongly either way!